### PR TITLE
worked on cart and wishlist functionality

### PR DESCRIPTION
**Pull request by kgajriaAgarwal**
Added cart and wishlist functionality.Kindly review it and let me know what do you think about it!
preview : https://inspiring-daffodil-9b8dc0.netlify.app/login

### CODE REVIEW

**Review by sri-rishi — APPROVED** [review]
Code looks good, just one suggestion if not necessary remove comments, feel free to merge

File: src/Api/useAxios/useAxios.jsx [hunk 1]
```diff
@@ -1,33 +1,62 @@
-// useAxios hook
+// // useAxios hook
 
-import { useState, useEffect } from 'react';
+// import { useState, useEffect } from 'react';
+// import axios from 'axios';
+
```
Comment (MayankKumar10) [inline src/Api/useAxios/useAxios.jsx:5]: remove the commented code

File: src/Pages/Authentication/Login/Login.jsx [hunk 2]
```diff
@@ -20,10 +24,20 @@ const Login = () =>{
...
-        e.preventDefault();
+    const handleLogin = async (e) =>{
         //commented right now for future perspective..
-        // console.log('LoginData', loginData);
+        //console.log('LoginData', loginData);
+        try {
+            e.preventDefault();
+            const response = await axios.post("/api/auth/login",  loginData);
+            if (response.status === 200) {
+            console.log("responese", response);
```
Comment (MayankKumar10) [inline src/Pages/Authentication/Login/Login.jsx:34]: remove the console log as well

**Review by MayankKumar10 — APPROVED** [review]
Look into these suggestions else everything looks good to me you can merge it.

File: src/Api/useAxios/useAxios.jsx (hunk 1 above)
Comment (kgajriaAgarwal) [reply src/Api/useAxios/useAxios.jsx:5]: ok!

File: src/Pages/Authentication/Login/Login.jsx (hunk 2 above)
Comment (kgajriaAgarwal) [reply src/Pages/Authentication/Login/Login.jsx:34]: ok!

**Comment by j24m** [comment]
image

UI looks amazing just one thing, when I hover on the avatar on the nav a horizontal scroll at the bottom appears, and the dropdown menu overflows.

**Review by j24m — APPROVED** [review]

**Pull request merged**